src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.NoArmIntrinsics.xml

```diff
@@ -0,0 +1,31 @@
+<linker>
+  <assembly fullname="System.Private.CoreLib">
+    <type fullname="System.Runtime.Intrinsics.Arm.AdvSimd">
+      <method signature="System.Boolean get_IsSupported()" body="stub"/>
```

Comment:
Does lack of a `value` mean it defaults to `default(T)` and thus `false` for such Booleans?

Reply:
Correct.

Reply:
Should we specify the `value="false"` regardless so it's clear what the substitution is? I was confused for a second before I saw this comment.

Reply:
Done.